### ISSUE TITLE
Add TT_TestNullAndWrongTypeParams() tests to CASFRI helper function test file

### ISSUE THREAD

**Issue by edwardsmarc** [edwardsmarc/CASFRI#132]

**Comment by edwardsmarc** [comment]
@pedrogit 
Does TT_TestNullAndWrongTypeParams() only work when there is **one** source value?

In most of the custom functions  there are multiple input source values. Maybe I can just write individual test for this file.

**Comment by pedrogit** [comment]
It works only for functions that have one source value and some parameters.

**Comment by pedrogit** [comment]
I don't think we want to copy this function to CASFRI. CASFRI is already dependent on the translation engine and this function is part of its test part. Normally when you have tested the engine, the function becomes available.
